src/modules/v1/auth/strategies/jwt.strategy.ts

```diff
@@ -12,7 +13,7 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
   ) {
     super({
       jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
-      secretOrKey: configService.get('SECRET'), // replace with your own secret
+      secretOrKey: authConstants.jwt.secret, // replace with your own secret
```

Comment:
configService fetches from the environment variable, you might want to stick to using a .env file.